reinforcement_learning/trainer/task.py

```diff
@@ -0,0 +1,226 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl import app
+from absl import flags
+from c2a2_agent import C2A2
+from ddpg_agent import DDPG
+from td3_agent import TD3
+import gym
+from gym import wrappers
+import json
+import numpy as np
+import os
+import tensorflow as tf
+from common import util
+
+
+FLAGS = flags.FLAGS
```

Comment:
use argparse instead.

Reply:
fixed in 994e1c88e71fa20ca85eb6ed2b81e816f8e5c475